site/content/contribute/why-contribute/_index.md

```diff
@@ -0,0 +1,99 @@
+        1. When contributions are inactive but for good reason (for example, when the team is actively discussing a major design decision but they haven’t decided on anything yet), `lifecycle/frozen` would be a better label.
+        2. Inactive contributions are eligible to be picked up by another community member.
+    3. After 30 days of inactivity, a contribution becomes orphaned, and they’ll add the `lifecycle/3:orphaned` label to the now-closed PR. The associated `Help Wanted` ticket is given back its `Up For Grabs` status so others can pick up on the issue.
+
+## Subsection 2 — You want to help with content
+
+1. Good content is just as important as good code! If you notice and fix a content error in the documentation or in another open source article describing Mattermost, we consider you to be a valued member of our contributor community just like if you had added a feature to the core code.
+2. If you see a problem with Mattermost [developer](https://developers.mattermost.com/) or [user](https://docs.mattermost.com/) documentation, you have a few options:
+    1. If you have time to fix the mistake and it only affects a single page, navigate to the applicable page and click “Edit in GitHub” at the top right.  It’ll walk you through the process of creating a fork so that you can then follow the steps under section 1, subsection 1 of this guide, “You’ve found a bug”.
+    2. If you don’t have time to fix the mistake, copy the file path you’re on and create a GitHub issue about the problem you found. Make sure to include the file path and fill out the issue template completely to maximize clarity.
+    3. If you’re not up for creating a GitHub issue right now, that’s alright too! In the bottom right corner of every docs page is the question “Did you find what you were looking for?” Use this to quickly provide direct feedback about any page you’re viewing.
+    4. If you want to fix a larger problem that affects multiple pages or the structure of the docs, you should first report it as an issue on the appropriate GitHub repo, and follow the steps under section 1, subsection 1 of this guide, “You’ve found a bug”. The developer and user docs repositories contain instructions on how to build and modify the sites locally so you can test larger changes more efficiently.
+    5. The best place to discuss problems with us is in the [Documentation Working Group channel](https://community.mattermost.com/core/channels/dwg-documentation-working-group) where you can ping our technical writers with the hashtag #docsteam.
```

Comment:
Is it `#docsteam` or `@docsteam`? I'm 3/5 on `@docsteam`

Reply:
The Technical Writing team has a user group called ``@docsteam``. @neflyte - Are you open to updating this PR with this correction?